src/components/image.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { parseSize } from '../utils';
+import Wrapper from './wrapper';
+
+export default
+class AvatarImage extends React.PureComponent {
+
+    static propTypes = {
+        alt: PropTypes.string,
+        title: PropTypes.string,
+        name: PropTypes.string,
+        value: PropTypes.string,
+        avatar: PropTypes.object,
+
+        className: PropTypes.string,
+        unstyled: PropTypes.bool,
+        round: PropTypes.oneOfType([
+            PropTypes.bool,
+            PropTypes.string
+        ]),
+        size: PropTypes.oneOfType([
+            PropTypes.number,
+            PropTypes.string
+        ]),
+    }
+
+    static defaultProps = {
+        className: '',
+        round: false,
+        size: 100,
+        unstyled: false
+    }
+
+    render() {
+        const {
+            className,
+            round,
+            unstyled,
+            alt,
+            title,
+            name,
+            value,
+            avatar
+        } = this.props;
+
+        const size = parseSize(this.props.size);
+
+        const imageStyle = unstyled ? null : {
+            maxWidth: '100%',
+            width: size.str,
+            height: size.str,
+            borderRadius: (round === true ? '100%' : round)
```

Comment:
won't this pass **round** as borderRadius if it is false?

Reply:
If you pass `false` here, borderRadius will simply be removed from the final stylesheet so this doesn't pose a problem